templates/app/models/index.hbs

```diff
@@ -1,97 +1,98 @@
-  process.exit();
-}
-
-const databaseOptions = {
-  logging: process.env.NODE_ENV === 'development' ? console.log : false,
-  pool: { maxConnections: 10, minConnections: 1 },
-  dialectOptions: {},
-};
-
-{{#if isMySQL}}
-databaseOptions.dialectOptions.typeCast = (field, useDefaultTypeCasting) => {
-  if ((field.type === "BIT") && (field.length === 1)) {
-    const bytes = field.buffer();
-    return bytes ? bytes[0] === 1 : bytes;
-  }
-
-  return useDefaultTypeCasting();
-};
-
 {{/if}}
-if (process.env.DATABASE_SSL && JSON.parse(process.env.DATABASE_SSL.toLowerCase())) {
-{{#if isMySQL}}
-  databaseOptions.dialectOptions.ssl = { rejectUnauthorized: true };
-{{else if isMSSQL}}
-  databaseOptions.dialectOptions.options = { encrypt: true };
-{{else}}
-  const rejectUnauthorized = process.env.DATABASE_REJECT_UNAUTHORIZED;
-  if (rejectUnauthorized && (JSON.parse(rejectUnauthorized.toLowerCase()) === false)) {
-    databaseOptions.dialectOptions.ssl = { rejectUnauthorized: false };
-  } else {
-    databaseOptions.dialectOptions.ssl = true;
+
+const databasesConfiguration = [{
+  name: 'default',
+  modelsDir: '.',
+  connection: {
+    url: process.env.DATABASE_URL
   }
-{{/if}}
-}
+}];
 
-const sequelize = new Sequelize(process.env.DATABASE_URL, databaseOptions);
+const connections = {};
 const db = {};
 
-fs
-  .readdirSync(__dirname)
-  .filter((file) => {
-    return (file.indexOf('.') !== 0) && (file !== 'index.js');
-  })
-  .forEach((file) => {
-    try {
-      const model = sequelize.import(path.join(__dirname, file));
-      db[model.name] = model;
-    } catch (error) {
-      console.error('Model creation error: ' + error);
+databasesConfiguration.forEach((databaseInfo) => {
+  const databaseOptions = {
+    {{#if isMongoDB}}
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    {{else}}
+    logging: !process.env.NODE_ENV || process.env.NODE_ENV === 'development' ? console.log : false,
+    pool: { maxConnections: 10, minConnections: 1 },
+    dialectOptions: {},
+    {{/if}}
+  };
+  {{#if isMySQL}}
+  databaseOptions.dialectOptions.typeCast = (field, useDefaultTypeCasting) => {
+    if ((field.type === "BIT") && (field.length === 1)) {
+      const bytes = field.buffer();
+      return bytes ? bytes[0] === 1 : bytes;
+    }
+
+    return useDefaultTypeCasting();
+  };
+  {{/if}}
+  {{#unless isMongoDB}}
+  if (process.env.DATABASE_SSL && JSON.parse(process.env.DATABASE_SSL.toLowerCase())) {
+    {{#if isMySQL}}
+    databaseOptions.dialectOptions.ssl = { rejectUnauthorized: true };
+    {{else if isMSSQL}}
+    databaseOptions.dialectOptions.options = { encrypt: true };
+    {{else}}
+    const rejectUnauthorized = process.env.DATABASE_REJECT_UNAUTHORIZED;
+    if (rejectUnauthorized && (JSON.parse(rejectUnauthorized.toLowerCase()) === false)) {
+      databaseOptions.dialectOptions.ssl = { rejectUnauthorized: false };
+    } else {
+      databaseOptions.dialectOptions.ssl = true;
     }
-  });
+{{/if}}
```

Comment:
[Bug] Appear to be a bug in the handlebars library. Maintaining correct indentation here breaks the generation

Reply:
https://github.com/handlebars-lang/handlebars.js/issues/1716